packages/flutter/lib/src/services/sensitive_content.dart

```diff
@@ -0,0 +1,81 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// @docImport 'package:flutter/widgets.dart';
+library;
+
+import 'package:flutter/foundation.dart';
+
+import 'system_channels.dart';
+
+/// The level of sensitivity that content in a particular Flutter view
+/// contains.
+/// 
+/// * See [SensitiveContent] for how to set a [ContentSensitivity] level
+///   in order for sensitive content to be obscured when the Flutter screen
+///   is shared. 
+enum ContentSensitivity {
+  /// Content sensitivity is auto-detected by the native framework.
+  /// 
+  /// When this level is set via a [SensitiveContent] widget, the window
+  /// hosting the screen will only be marked as secure if other [SensitiveContent]
+  /// widgets with the [sensitive] level are present in the widget tree.
+  /// 
+  /// For Android, the sensitive content is unable to be auto-detected by the
+  /// native framework.
```

Comment:
Nit: Maybe specify that you mean within the Flutter app. Also, this might be more complicated if we add a SensitiveContent widget when a user uses a password field. I guess that's what the TODO below is though.